utils/pod/pod.go

```diff
@@ -763,6 +765,25 @@ func SendMesosStatus(driver executor.ExecutorDriver, taskId *mesos.TaskID, state
 		TaskId: taskId,
 		State:  state,
 	}
+
+	logStatus := waitUtil.GetLogStatus()
+	log.Printf("Log status is : %v", logStatus)
+	log.Printf("Task status is : %v", state.Enum().String())
+
+	if logStatus == true {
+		if state.Enum().String() == mesos.TaskState_TASK_FAILED.Enum().String() ||
```

Comment:
remove TaskState_TASK_FAILED here, since log cmd may fail. Or we only have 3 retry if it's TaskState_TASK_FAILED. don't retry forever